calcom/cal.com
# fixes available times overflow bug on safari

## What does this PR do?

Fixes overflow bug on Safari, where there was too much space between available times.

Fixes #6201 



**Environment**: Production

## Type of change


## How should this be tested?

* Make an event type with a duration of 90 minutes
* Load in mobile Safari or desktop safari in really small window
* Notice that the space between available times is back to normal

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
